cdap-app-fabric/src/main/java/io/cdap/cdap/metadata/FieldLineageAdmin.java

```diff
@@ -384,4 +384,15 @@ private Set<Field> filter(String prefix, Set<Field> fields) {
                                     field.getName().toLowerCase().startsWith(prefix.toLowerCase()))
       .collect(Collectors.toSet());
   }
+
+  /**
+   * Retrieves list of all endpoints in a given run of a pipeline.
+   *
+   * @param namespaceId pipeline namespace
+   * @param startMillis start time of the run in millis corresponding to the id of a pipeline run
```

Comment:
shouldn't this be taking the run id and not the start time? We want to get endpoints for a particular run right?

Reply:
We were deriving starttime from the run id. But now this logic is gone and instead we are passing the program run id